cmd/template/cluster/flag.go

```diff
@@ -127,9 +124,7 @@ func (f *flag) Validate() error {
 
 	var release *gsrelease.GSRelease
 	{
-		c := gsrelease.Config{
-			NoCache: f.NoCache,
-		}
+		c := gsrelease.Config{}
```

Comment:
The branch name is configurable. Command flags can now easily be added. Not doing this here now though.